TeaDiary/Scripts/App/teaDisplayer.js

```diff
@@ -0,0 +1,32 @@
+﻿function TeaDisplayer(containers) {
```

Comment:
Please decide if you want to use `PascalCase` or `camelCase` in your file names and stick to it!